grakn-test/test-integration/src/test/bash/build-grakn.sh

```diff
@@ -18,4 +18,4 @@ echo "Setting version using branch name"
 mvn versions:set -DnewVersion=${BRANCH_NAME} -DgenerateBackupPoms=false
 
 echo "Installing grakn"
-mvn clean install --batch-mode -Dmaven.repo.local=${WORKSPACE}/maven -DskipTests -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT
+mvn clean install -T 14 --batch-mode -Dmaven.repo.local=${WORKSPACE}/maven -DskipTests -U -Djetty.log.level=WARNING -Djetty.log.appender=STDOUT
```

Comment:
Can we turn this up to 15 please?

Also uh what does this number do

Reply:
ahem, 14 is a lot maybe but it's the number of thread per core used to build. If we are not testing that doesn't cause problems

Reply:
I'll reduce it on the next one and merge it then!